versionutils/version.go

```diff
@@ -93,3 +93,33 @@ func MatchesRegex(tag string) bool {
 	regex := regexp.MustCompile("(v[0-9]+[.][0-9]+[.][0-9]+$)")
 	return regex.MatchString(tag)
 }
+
+func IncrementVersion(v *Version, breakingChange bool) *Version {
```

Comment:
Could make patch, minor, and major functions so this package could be used elsewhere, not just by changelog